src/runtime/offers-flow.js

```diff
@@ -238,7 +239,10 @@ export class AbbrvOfferFlow {
     // If result is due to requesting offers, redirect to offers flow
     this.activityIframeView_.acceptResult().then(result => {
       if (result.data['viewOffers']) {
-        new OffersFlow(this.deps_, this.options_).start();
+        const isClosable = result.data && result.data['isClosable'] || false;
+        const options = this.options_ || {};
+        Object.assign(options, {'isClosable': isClosable});
```

Comment:
This is not a string property - it's a field. I'm afraid you'll have to programmatically construct options.

Reply:
Done